tools/disasm/functions.txt

```diff
@@ -16644,9 +16644,9 @@
     0x80C09AA4:("func_80C09AA4",),
     0x80C09B50:("func_80C09B50",),
```

Comment:
Looks like there are some functions that are named in the C file that aren't named in `functions.txt`. Would be good to keep this up-to-date while we're naming these functions.